content/blog/example-of-mapreduce-with-ezmeral-data-fabric-mapr-database-binary-table.md

```diff
@@ -0,0 +1,365 @@
+---
+title: Example of MapReduce With Ezmeral Data Fabric(MapR) Database Binary Table
+date: 2022-11-14T15:56:03.596Z
+featuredBlog: false
+author: Raymond Yan
+authorimage: /img/raymondyan-profile.png
+thumbnailimage: /img/mapreduce_with_edf_binarytable.png
+disable: false
+tags:
+  - hpe-ezmeral-data-fabric
+  - MapReduce
+---
+
+## Introduction
+
```

Comment:
Dale will help you with the introduction section to ensure that there's a hook  at the beginning, something which will draw the reader in.